### PR TITLE
Associated Service Worker Registration for CookieStoreManager

### PR DESCRIPTION
**Pull request by ayuishii**
Issue: https://github.com/WICG/cookie-store/issues/180

***
<a href="https://pr-preview.s3.amazonaws.com/ayuishii/cookie-store/pull/181.html" title="Last updated on Sep 17, 2020, 12:36 AM UTC (b1ca92d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/cookie-store/181/450c023...ayuishii:b1ca92d.html" title="Last updated on Sep 17, 2020, 12:36 AM UTC (b1ca92d)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -913,7 +915,7 @@ The <dfn method for=CookieStoreManager>unsubscribe(|subscriptions|)</dfn> method
 ## The {{ServiceWorkerRegistration}} interface ## {#ServiceWorkerRegistration}
 <!-- ============================================================ -->
 
-The {{ServiceWorkerRegistration}} interface is extended to give access to a {{CookieStoreManager}}.
+The {{ServiceWorkerRegistration}} interface is extended to give access to a {{CookieStoreManager}} via {{cookies}} which provides the interface for subscribing to cookie changes.
 
 <xmp class=idl>
 [Exposed=(ServiceWorker,Window)]
```
Comment (mkruisselbrink) [inline index.bs:921]: kind of separate to this PR, but cookies should probably be [SameObject]. And the cookies attribute should normatively declare what it returns somewhere, i.e. something like this:

```
Each {{ServiceWorkerRegistration}} has an associated {{CookieStore}} object.
The {{CookieStore}}'s [=CookieStore/registration=] is equal to the {{ServiceWorkerRegistration}}'s [=ServiceWorkerRegistration/service worker registration=].

The {{ServiceWorkerRegistration/cookies}} [=getter steps=] are to return [=this=]'s associated {{CookieStore}} object.
```

Comment (ayuishii) [reply index.bs:921]: Thanks for the guidance! I made the change to SameObject and added the normative bit below. Made some changes to get the links to work, but let me know if I've missed anything.

Comment (mkruisselbrink) [reply index.bs:921]: Huh, I see. Technically it really should be [=ServiceWorkerRegistration/service worker registration=], but I guess that term isn't exported by the service worker spec, so what you have now seems good enough.
(as background, [=service worker registration=] would link to https://w3c.github.io/ServiceWorker/#dfn-service-worker-registration, while what we really want here is https://w3c.github.io/ServiceWorker/#serviceworkerregistration-service-worker-registration, but as said that term isn't exported. It's not like it isn't clear what we're talking about so the current text seems fine to me).

**Review by mkruisselbrink — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (ayuishii) [reply index.bs:921]: I see, just out of curiosity how can you tell if a term has been exported or not?

**Comment by ayuishii** [comment]
Thank you for the review!

**Pull request merged**